data_models/data_refinery_models/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from django.utils import timezone
+from data_refinery_models.models import SurveyJob
+
+
+class TimeTrackedModelTestCase(TestCase):
+    def setUp(self):
+        SurveyJob.objects.create(source_type="ARRAY_EXPRESS")
+
+    def tearDown(self):
+        SurveyJob.objects.all().delete()
```

Comment:
But it only removes it when all the tests have completed. By doing this I ensure that if I write any other tests they won't interfere.